docs/INSTALLATION.md

```diff
@@ -64,6 +64,20 @@ of the Microsoft Visual C++ compiler.
 
 You should be able to use other integrated IDE's such as JetBrains' CLion in a similar manner.
 
+### Installing Boost on Windows
+
+The `meetup` and `gigasecond` exercises depend on the [Boost][boost-website] libraries.
+The following steps describe how to set that up.
+
+1. Download the [prebuilt windows libraries][sourceforge-boost-binaries] for Boost.
```

Comment:
The enumeration is picked up correctly in the rich-view in github, but here every item is labeled as number 1. I am surprised that this works anyhow. :confused:

Reply:
@vaeng Please not that the same document [already uses this syntax](https://github.com/exercism/cpp/blob/main/docs/INSTALLATION.md?plain=1#L63).

It seems there is no formal markdown spec, but there is e.g. [this](https://commonmark.org/), [which states](https://spec.commonmark.org/0.31.2/#lists):

> The [start number](https://spec.commonmark.org/0.31.2/#start-number) of an [ordered list](https://spec.commonmark.org/0.31.2/#ordered-list) is determined by the list number of its initial list item. The numbers of subsequent list items are disregarded.

Using `1.` for all items eases maintenance when adding, removing or reordering items and helps with diffs.